C++ compilation gets an error when you run it for Japanese version of Microsoft Visual Studio 2019

### Expected Behavior
It should compile a source code.

### Current Behavior
Any try of compilation gets the following error.
/*
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 't'.
> com.google.gson.stream.MalformedJsonException: Invalid escape sequence at line 16 column 39 path $[0].description

* Try:
Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Exception is:
... (omitted) ...
Caused by: com.google.gson.stream.MalformedJsonException: Invalid escape sequence at line 16 column 39 path $[0].description
	at com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1568)
	at com.google.gson.stream.JsonReader.readEscapeCharacter(JsonReader.java:1559)
	at com.google.gson.stream.JsonReader.skipQuotedValue(JsonReader.java:1109)
	at com.google.gson.stream.JsonReader.skipValue(JsonReader.java:1252)
	at org.gradle.nativeplatform.toolchain.internal.msvcpp.version.CommandLineToolVersionLocator.readInstall(CommandLineToolVersionLocator.java:129)
	at org.gradle.nativeplatform.toolchain.internal.msvcpp.version.CommandLineToolVersionLocator.parseJson(CommandLineToolVersionLocator.java:100)
	... 147 more
 */

### Context
<!--- How has this issue affected you? What are you trying to accomplish? -->
<!--- Providing context helps us come up with a solution that is most useful in the real world -->

### Steps to Reproduce 
"gradle tasks" command with build.gradle below:

plugins {id 'cpp-application'}
version = '1.2.1'
application {
}
### Your Environment
Microsoft Visual Studio 2019 Japanese version
Windows 10 Home edition Japanese version

### Fix up
If you apply the following change, it works perfectly.
org.gradle.nativeplatform.toolchain.internal.msvcpp.version.CommandLineToolVersionLocator(Line: 58)
before: List<String> args = Lists.newArrayList("-all", "-legacy", "-format", "json");
    after: List<String> args = Lists.newArrayList("-all", "-legacy", "-format", "json", "-utf8");

It seems the default output character code of Microsoft Visual Studio/Installer/vswhere.exe is MS932(Shift_JIS) rather than UTF-8 unless "-utf8" is specified.

--- a/subprojects/platform-native/src/main/java/org/gradle/nativeplatform/toolchain/internal/msvcpp/version/CommandLineToolVersionLocator.java
+++ b/subprojects/platform-native/src/main/java/org/gradle/nativeplatform/toolchain/internal/msvcpp/version/CommandLineToolVersionLocator.java
@@ -55,7 +55,7 @@ public class CommandLineToolVersionLocator extends AbstractVisualStudioVersionLo
 
         File vswhereBinary = vswhereLocator.getVswhereInstall();
         if (vswhereBinary != null) {
-            List<String> args = Lists.newArrayList("-all", "-legacy", "-format", "json");
+            List<String> args = Lists.newArrayList("-all", "-legacy", "-format", "json", "-utf8");
             String json = getVswhereOutput(vswhereBinary, args);
             installs.addAll(parseJson(json));
         }

